.dockerignore

```diff
@@ -1,2 +1,20 @@
-# Ignore everything
-*
\ No newline at end of file
+abc
```

Comment:
This file is to guarantee that docker will not include an old version of OpenFPGA, right? Just to confirm.

Reply:
The dockerignore is used to limit which files are mounted in the docker context when building a container image. @ganeshgore introduced the original file, I'm not sure what his original plan was. I replaced the blanket ignore all with ignoring everything but the requirements.txt file so I could run pip in the build.

Reply:
This is important to reduce docker image size, you should add all files/directories which doesn't affect your docker image. The current content looks good